azure/blobs_logs_monitoring/index.js

```diff
@@ -41,7 +41,7 @@ module.exports = function(context, blobContent) {
         }
     };
 
-    var logs = blobContent.trim().split('\n');
+    var logs = blobContent.toString().trim().split('\n');
```

Comment:
I think what you want is because of the image below

```suggestion
    var logs;
    if (typeof blobContent === 'string') {
        logs = blobContent.trim().split('\n');
    } else {
        logs = JSON.stringify(blobContent).trim().split('\n');
    }
```

![image](https://user-images.githubusercontent.com/6985756/57407392-57704080-71e3-11e9-947a-83573c7e241f.png)